self-core-impl/src/test/java/com/selfxdsd/core/StoredUserTestCase.java

```diff
@@ -148,4 +148,49 @@ public void returnsProjects() {
         );
     }
 
+    /**
+     * Can compare two StoredUser objects.
+     */
+    @Test
+    public void comparesStoredUserObjects() {
+        final User user = new StoredUser(
+            "amihaiemil",
+            "amihaiemil@gmail.com",
+            "user",
+            Provider.Names.GITHUB,
+            Mockito.mock(Storage.class)
+        );
+        final User userTwo = new StoredUser(
+            "amihaiemil",
+            "amihaiemil@gmail.com",
+            "user",
+            Provider.Names.GITHUB,
+            Mockito.mock(Storage.class)
+        );
+        MatcherAssert.assertThat(user, Matchers.is(userTwo));
```

Comment:
@ANadrowski Can you use ``Matchers.equalTo(...)``? I think the ``is(...)`` method compares references only.

Reply:
@amihaiemil it's ok, I am going to implement it soon :)